dask_expr/_expr.py

```diff
@@ -2603,6 +2603,56 @@ def _fusion_pass(expr):
     return expr
 
 
+class Fill(MapOverlap):
```

Comment:
FFill is a better name

Reply:
https://github.com/dask-contrib/dask-expr/pull/454/commits/0b201d8a143e4b0981d32ef5e3ab52366fea5b2b